libminifi/include/core/ProcessContextBuilder.h

```diff
@@ -40,6 +40,7 @@
 #include "core/Repository.h"
 #include "core/FlowFile.h"
 #include "VariableRegistry.h"
+#include "utils/gsl.h"
```

Comment:
I think this include is not necessary, probably a leftover from a previous development version.

Reply:
Good catch, this is now removed.